src/App.js

```diff
@@ -1,9 +1,15 @@
+import React, { Component } from 'react';
 import './App.css';
+import Person from './Person/Person.js';
```

Comment:
There is an approach in the React Style Guide to get rid of repeatable naming in the imports - by following component naming best practices:
https://github.com/airbnb/javascript/tree/master/react#naming
![image](https://user-images.githubusercontent.com/52723192/103827497-316f0380-508a-11eb-9615-bded9442b91e.png)


Please rename component and css files correspondingly.